src/feedback.tsx

```diff
@@ -196,6 +201,7 @@ export const FeedbackScreen: React.FC<FeedbackScreenProps> = ({
     title,
     description,
     children,
```

Comment:
deprecated?

Reply:
It's marked as deprecated in the type declaration :)